less/viewdetails.less

```diff
@@ -606,6 +606,10 @@
       color: @C2C-orange
     }
   }
+
```

Comment:
To be consistent with the other blocks? Except maybe ``.area-associations`` just above, there are blank lines before selector blocks.